pkg/catalogsourceconfig/registry.go

```diff
@@ -243,27 +242,35 @@ func (r *registry) getLabel() map[string]string {
 	return map[string]string{"marketplace.catalogSourceConfig": r.csc.GetName()}
 }
 
-// getAppRegistryCmdLineOptions returns a group of "--registry=" command line
-// option(s) required for operator-registry. If one of the packages is from a
-// private repository secretIsPresent will be true.
-func (r *registry) getAppRegistryCmdLineOptions() (appRegistryOptions string, secretIsPresent bool) {
+// getAppRegistries returns a list of app registries in the format
+// {base url with cnr prefix}|{quay registry namespace}|{secret namespace/secret name}.
+// |<secret namespace/secret name} will be present only for private repositories,
+// in which case secretIsPresent will be true.
+func (r *registry) getAppRegistries() (appRegistries []string, secretIsPresent bool) {
 	for _, packageID := range r.csc.Spec.GetPackageIDs() {
 		opsrcMeta, err := r.reader.Read(packageID)
 		if err != nil {
 			r.log.Errorf("Error %v reading package %s", err, packageID)
 			continue
 		}
-		//--registry="https://quay.io/cnr|community-operators" --regisry="https://quay.io/cnr|custom-operators|mynamespace/mysecret"
-		appRegistry := "--registry=" + opsrcMeta.Endpoint + "|" + opsrcMeta.RegistryNamespace
+		// {base url with cnr prefix}|{quay registry namespace}
```

Comment:
I know the old comment specifically called out Quay, but maybe this comment should just call it `app registry namespace` instead? Technically this particular pattern could be used for any `app-registry` that the `operator-registry` implemented

Reply:
I copied this from https://github.com/operator-framework/operator-registry/commit/717cba98d4254ea5983579a5370354653ecfa553#diff-1577e3efd6ba0d0bb82f2e00c849137aR37. But you do have a point. @tkashem this should be fixed there too.